spec/requests/admin/color_schemes_controller_spec.rb

```diff
@@ -31,6 +31,20 @@
         expect(scheme_colors[0]["name"]).to eq(base_scheme_colors[0].name)
         expect(scheme_colors[0]["hex"]).to eq(base_scheme_colors[0].hex)
       end
+
+      it "serializes default colors even when not present in database" do
+        scheme = ColorScheme.create_from_base({ name: "my color scheme" })
+        scheme.colors.find_by(name: "primary").destroy
```

Comment:
```suggestion
        scheme.colors.find_by(name: "primary").destroy!
```

There could be linter rule/error: "you're not checking the return value of `#destroy`, add a result check or use `#destroy!`" 😁